src/components/hero/Hero.tsx

```diff
@@ -0,0 +1,59 @@
+import { Col, Image } from "@nextui-org/react";
+import { StyledContainer, StyledSubtitle, StyledTitle } from "./Hero.styles";
```

Comment:
TypeScript has been configured with a path alias, so can you be explicit about where you're importing the `Hero.styles` module from?

For more context, use this syntax:
```tsx
import { StyledContainer, ... } from "@components/hero/Hero.styles"
```

Being explicit makes reading the source code much easier, else I've to take a second look at the file explorer to check where was it imported from.